src/app/containers/OnDemandHeading/index.jsx

```diff
@@ -17,16 +26,29 @@ const BrandTitle = styled.span`
   word-break: break-word;
   ${({ script, darkMode }) => (darkMode ? '' : script && getParagon(script))}
   ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
-    padding-bottom: 0;
+    padding-bottom: ${GEL_SPACING_DBL};
     word-break: break-word;
-    line-height: ${GEL_SPACING_SEPT};
+    line-height: ${GEL_SPACING_QUAD};
   }
 `;
 
 const Subheading = styled.span`
+  display: inline-block;
+  margin: 0;
   ${({ script }) => script && getDoublePica(script)}
   ${({ service }) => getSansRegular(service)}
-  margin: 0;
+  ${MEDIA_QUERY_TYPOGRAPHY.LAPTOP_AND_LARGER} {
```

Comment:
This is really cool, I wasn't aware of this feature of gel-foundations where it gives us the actual media queries 🙂 Although inspecting the package, think this will give us `@media (min-width: 600px)` and I can't find a breakpoint in the designs for > 600px?

Reply:
**We haven't been given a specification for breakpoint >600px but have requested this from UX so will address if appropriate. Thanks!**